multimodal/src/autogluon/multimodal/data/process_categorical.py

```diff
@@ -89,6 +89,7 @@ def __call__(
         categorical_features: Dict[str, int],
         feature_modalities: Dict[str, Union[int, float, list]],
         is_training: bool,
+        load_only: bool = False,
```

Comment:
Do we need to add this flag in each processor class? How about restricting it only to the related processors and using an if condition when calling processors in `apply_data_processor`?

Reply:
Done. I was thinking that later we may need to use it in other processors as well. But we can add when necessary.